documentation/contributing/releases/index.markdown

```diff
@@ -0,0 +1,111 @@
+---
+author: tylerjw
+layout: page
+slug: releases
+title: Releases
+---
+
+# Release Process
+
+MoveIt has a 6 week cadence for releases.  This pace is a balance between releasing new improvements quickly and preserving the stability of releases.  We are choosing this schedule to reduce the amount of work per release, reduce the time it takes to get an improvement released and make it easier to schedule for the Picknik team.  These releases will include the main MoveIt repo along with any dependencies we control if they have changed (e.g. moveit_msgs).
+
+## Upcoming releases
+
+The release schedule is avalible via [this Google Calendar](https://calendar.google.com/calendar/embed?src=c_bpueq5jcajuaejr2di0hdr4qgg%40group.calendar.google.com&ctz=America%2FDenver).
```

Comment:
rather than href, can you grab the html from the calendar such that it iframes? 
https://support.google.com/calendar/answer/41207?hl=en

markdown supports arbitrary html

Reply:
```suggestion
The release schedule is shown here:

<iframe src="https://calendar.google.com/calendar/embed?src=c_bpueq5jcajuaejr2di0hdr4qgg%40group.calendar.google.com&ctz=America%2FDenver" width="100%" height="350px" title="MoveIt release calendar"></iframe>
```